### PR TITLE
A levels - Add a levels equivalency tests

### PR DESCRIPTION
**Pull request by tomas-stefano**
### Context

We need to add the ability for providers to answer if they will consider candidates who need to take equivalency tests to TDA a course on Publish, in order to enable TDA applications.

### Changes proposed in this pull request

Add new page

Screenshot 2024-06-26 at 19 12 59

### Guidance to review

1. All validations work?
2. Are you happy with field names? (in the step, the name of the attributes of the step and the DB fields?)
3. Does it work if you return to the page after answering do you see your answer?

What this PR does not contain?

* Showing the information on the course description (this will be the part 7 card in Trello)

In order to see locally you need to enable the rollover and be on 2025 cycle when you click around in Publish.

## Trello card

https://trello.com/c/08UW62iD/1791-a-levels-6-7-add-a-levels-equivalency-tests-question-page-for-tda-courses-on-publish

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[A levels 6/7 - Add A levels equivalency tests question page for TDA courses on Publish](https://trello.com/c/08UW62iD/1791-a-levels-6-7-add-a-levels-equivalency-tests-question-page-for-tda-courses-on-publish)

File: db/migrate/20240626152030_add_accept_a_level_equivalencies_to_course.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddAcceptALevelEquivalenciesToCourse < ActiveRecord::Migration[7.1]
+  def change
+    add_column :course, :accept_a_level_equivalencies, :boolean
```
Comment (tomas-stefano) [inline db/migrate/20240626152030_add_accept_a_level_equivalencies_to_course.rb:3]: After I opened the PR I realise the other fields for GCSE and degrees are *_equivalency for this field. Let me know if I should change now.

Comment (tomas-stefano) [reply db/migrate/20240626152030_add_accept_a_level_equivalencies_to_course.rb]: Done

**Review by CatalinVoineag — COMMENTED** [review]
From where I am the code looks good.

This might be a product thing but maybe communicating to the user that he can put only 4 subjects would be helpful? Unless that's understood already

Screenshot from 2024-06-28 13-54-59


And another thing is more to do with the way the multistep form works. This might not be a problem.
If you abandon the form when adding A levels mid-way the change will persist on the course and you can then publish the course without adding a subject. It's quite an edge case


https://github.com/DFE-Digital/publish-teacher-training/assets/11318084/3832ed06-bb39-448a-ac6a-fa223a464a44

**Comment by tomas-stefano** [comment]
> This might be a product thing but maybe communicating to the user that he can put only 4 subjects would be helpful? Unless that's understood already

Yes, only the validation triggers. there is no content explain it. I will raise this :) - but probably will be in a future small card and PR.

> And another thing is more to do with the way the multistep form works. This might not be a problem.
If you abandon the form when adding A levels mid-way the change will persist on the course and you can then publish the course without adding a subject. It's quite an edge case

Yes, I raised already last week. I didn't create a Trello card for this though. I will do it know

**Review by CatalinVoineag — APPROVED** [review]
🚀

**Pull request merged**